packages/design-system/src/connect/components/home-screen/home-screen.tsx

```diff
@@ -0,0 +1,31 @@
+import { Icon } from "@/powerhouse";
+import { ComponentPropsWithRef, ForwardedRef, forwardRef } from "react";
+import { twMerge } from "tailwind-merge";
+import { HomeScreenItem } from "../home-screen-item";
+type InputProps = ComponentPropsWithRef<"input">;
```

Comment:
Since this is not an input there is no need to add this type.

Reply:
For both of these there is no need to use the component props of input. this makes things a lot more complicated by including more than 100 extra props. we can see from usage that these other props are not used or forwarded to other components, so they are redundant and give the impression that these props are doing something when they actually are not. 

In general its best to only define the props you actually intend to use. 

this is all you need here, for example

`
type Props = {
  children: React.ReactNode;
  containerClassName?: string;
}
`